packages/url_launcher/url_launcher_android/test/url_launcher_android_test.dart

```diff
@@ -0,0 +1,226 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:url_launcher_android/url_launcher_android.dart';
+import 'package:url_launcher_platform_interface/url_launcher_platform_interface.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  group('$UrlLauncherAndroid', () {
+    const MethodChannel channel =
+        MethodChannel('plugins.flutter.io/url_launcher_android');
+    final List<MethodCall> log = <MethodCall>[];
+    channel.setMockMethodCallHandler((MethodCall methodCall) async {
+      log.add(methodCall);
+
+      // Return null explicitly instead of relying on the implicit null
+      // returned by the method channel if no return statement is specified.
+      return null;
+    });
+
+    final UrlLauncherAndroid launcher = UrlLauncherAndroid();
```

Comment:
Minor nit: Please move this into each test. Having the object under test be part of the test fixture makes tests harder to understand, doesn't scale to tests that need to initialize things differently, and makes it *much* easier to accidentally have state bleed across tests (which I've seen cause tests to pass when they should have failed due to something that was set in a completely different test far more times than I would like).

While this object is so simple it may well never come up, it's an anti-pattern that I don't want to have (or at least, add new) examples of in the codebase. The risk is just not worth saving one statement per test.